apps/web/src/design-system/select/Select.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Box,
+  Select as MantineSelect,
+  MultiSelect as MantineMultiSelect,
+  ChevronIcon,
+  CloseButton,
+  InputBaseProps,
+  MultiSelectValueProps,
+  useMantineTheme,
+} from '@mantine/core';
+import CheckboxItem from './CheckboxItem';
+import useStyles from './Select.styles';
+import { inputStyles } from '../config/inputs.styles';
+
+interface ISelectProps {
+  data: (string | { value: string; label?: string })[];
+  label?: React.ReactNode;
+  placeholder?: string;
+  description?: string;
+  type?: 'Checkbox' | 'Label';
```

Comment:
@ainouzgali can we make the types lowercased? 
```suggestion
  type?: 'checkbox' | 'label';
```

Reply:
sure :) I think I'll also change it to multiselect | select(default) , wdyt?